example.py

```diff
@@ -23,19 +23,18 @@ def wrap_in_payload(crash_data):
     trace_dict = wrap_in_payload(json.loads(sample_traces.string_trace2))
 
     symbolicated = crash_processor.symbolicate(trace_dict)
-
     print(symbolicated)
 
     if symbolicated is not None:
-        signature = crash_processor.get_signature_from_symbolicated(symbolicated)
-        print(signature)
+        result = crash_processor.get_signature_from_symbolicated(symbolicated)
```

Comment:
siggen 0.2.0 changed the API so it returns a `Result` now. I adjusted the code accordingly.